enzy_htp/_config/gaussian_config.py

```diff
@@ -28,13 +32,54 @@ class GaussianConfig(BaseConfig):
         LAYER_PRESET : int() default ONIOM level for simulation.
         LAYER_ATOMS : list() holding the atom names.
     """
+    DEFAULT_SOLVENT_MODEL: str = "SMD"
+    """default solvent model when a solvent is specificed but not solvent model"""
 
-    G16_EXE: str = "g16"
+    GAUSSIAN_EXE: str = "g16"
     """Variable for the g16 version of Gaussian."""
 
-    G09_EXE: str = "g09"
-    """Variable for the g09 version of Gaussian."""
-
+    # region == Default values for build_single_point_engine() ==
+    DEFAULT_SPE_NAME: str = "gaussian_spe"
+    """The default value for the name tag of the SPE engine"""
+
+    DEFAULT_SPE_METHOD: QMLevelofTheory = QMLevelofTheory(
```

Comment:
I like this approach much more than before! It is modular and allows for a variety of settings to be given. The only change I would make is to add a `solvent_params` attribute which can maybe be a `dict()`. Often times packages allow you to specify solvent dielectric and solvent refractory index values. This allows you to manually specify any implicit solvent.

Reply:
Cool! My plan for the custom solvent support is to have it also in the solvent attribute. In this case, solvent can be a str that access some predefined parameters or a dictionary that have actual parameters.
A san check between consistency of parameters and the solvation model is also needed in the future.

Reply:
Addressed in level_of_theory.py